packages/ws-server/src/controllers/eth_sendRawTransaction.ts

```diff
@@ -0,0 +1,90 @@
+/* -
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2023-2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { sendToClient } from '../utils/utils';
+import { Relay } from '@hashgraph/json-rpc-relay';
+import { predefined } from '@hashgraph/json-rpc-relay';
+import jsonResp from '@hashgraph/json-rpc-server/dist/koaJsonRpc/lib/RpcResponse';
+
+/**
+ * Handles the "eth_sendRawTransaction" method request by submitting a raw transaction to the Websocket server.
+ * Validates the parameters, submits the transaction, and sends the txHash response back to the client.
+ * @param {any} ctx - The context object containing information about the WebSocket connection.
+ * @param {any} params - The parameters of the method request, expecting a single parameter: the signed transaction.
+ * @param {any} logger - The logger object for logging messages and events.
+ * @param {Relay} relay - The relay object for interacting with the Hedera network.
+ * @param {any} request - The request object received from the client.
+ * @param {string} socketIdPrefix - The prefix for the socket ID.
+ * @param {string} requestIdPrefix - The prefix for the request ID.
+ * @param {string} connectionIdPrefix - The prefix for the connection ID.
+ * @returns {Promise<any>} Returns a promise that resolves with the JSON-RPC response to the client.
+ * @throws {JsonRpcError} Throws a JsonRpcError if there is an issue with the parameters or an internal error occurs.
+ */
+export const handleEthSendRawTransaction = async (
+  ctx: any,
+  params: any,
+  logger: any,
+  relay: Relay,
+  request: any,
+  socketIdPrefix: string,
+  requestIdPrefix: string,
+  connectionIdPrefix: string,
+) => {
+  const SIGNED_TX = params[0];
+  const METHOD_NAME = 'eth_sendRawTransaction';
```

Comment:
nit: move out of this method into class so it's created once and reused

Reply:
pushed in the update.